.github/workflows/deploy-gcp-tests.yml

```diff
@@ -919,10 +919,182 @@ jobs:
             -e 'estimated progress.*current_height.*=.*2[0-9][0-9][0-9][0-9][0-9][0-9].*remaining_sync_blocks' \
             -e 'test result:.*finished in'
 
+  # follow the logs of the test we just launched, up to block 1,800,000 or later
+  # (or the test finishing)
+  logs-1800k:
```

Comment:
Just wondering why we're adding 3 new jobs here?
How long does each job take?

Our target time for each new job is 4-5 hours:
https://github.com/ZcashFoundation/zebra/blob/main/book/src/dev/continuous-integration.md#fixing-ci-sync-timeouts

Reply:
It was just following the previous increment. These shouldn't take anywhere close to 4-5 hours.